Automattic/jetpack
# CRM: Resolves 3104 - Emerald listviews

Resolves Automattic/zero-bs-crm#3104 - Emerald listviews

Upon merge, this will solve the following:
Automattic/zero-bs-crm#690
Automattic/zero-bs-crm#774
Automattic/zero-bs-crm#1571
Automattic/zero-bs-crm#2996
Automattic/zero-bs-crm#2997

## Proposed changes:


Old:
![image](https://github.com/Automattic/jetpack/assets/32492176/d794259a-c63b-4d67-b2a1-f6e3afd3874c)

New:
![image](https://github.com/Automattic/jetpack/assets/32492176/701daa99-5041-4bbd-b223-388312e523d6)

Notes:
* doesn't handle multiple segments of same name (pre-existing bug: Automattic/zero-bs-crm#3113)
* listview per-page settings are reset periodically (pre-existing bug: Automattic/zero-bs-crm#3115)
* scrolling issue on shorter listviews (pre-existing bug: Automattic/zero-bs-crm#3116)
* quickfilters sometimes are not available (pre-existing issue: Automattic/zero-bs-crm#1413)
* quickfilters aren't sorted (there currently is no way to differentiate between segment, status, and built-in filters)
* no top pagination (for long results pages)
* no bottom bulk actions dropdown (for long results pages)
* sort and pagination don't use AJAX calls (but quickfilters, tags, and search now do)
* no change in "no results" or "loading" content
* "table options" button should probably use our new button components
* list view options have not been touched
* there's no clear way to determine if a table field is a link
* listview field styles are not curated (see "Name and Avatar" column, for example)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Everything should function as before (or better). Mobile screen sizes are out of scope in this PR. Make sure to build CSS before testing.
